### DISCUSSION TITLE
Is the team's share ID safe to share around?

### DISCUSSION THREAD

**Discussion by chee** [local-first-web/auth#116] [Q&A]
i'm considering a process for being able to be active in different teams in different tabs in my app, and i'm wondering if it's a security risk to have the team's share ID in the url?

btw i love this library so much, thank you for it, it's a delight

**Comment by HerbCaudill** [comment]
Hi! A team's `shareId` is not a secret, it's fine to have it in the url.

**Reply by chee** [reply]
excellent news!

**Reply by HerbCaudill** [reply]
> btw i love this library so much, thank you for it, it's a delight

and thanks! I'd love to know more about what you're working on, feel free to email if you want to chat.
